github/resource_github_repository_file.go

```diff
@@ -3,7 +3,6 @@ package github
 import (
 	"context"
 	"log"
-	"net/http"
```

Comment:
What's with the deletes here?  Did you find a better way to achieve the same thing?

Reply:
Moved it to a different file because i needed those methods in my data source

Reply:
You think i should do it differently?

Reply:
Sorry didn't notice it was just a move.  Thought it had gone away and was confused as to how you were doing the same.